grammars/rust.cson

```diff
@@ -159,11 +159,46 @@
       { 'include': '#std_types' }
       { 'include': '#std_traits' }
       { 'include': '#type_params' }
+      {
+        'name': 'entity.name.type.rust'
+        'match': '\\b[A-Za-z]+\\b'
+      }
```

Comment:
Extract the block from here and place it after 144 line:
```cson
   'std_traits': {
     'comment': 'Standard library trait'
     'name': 'support.type.std.rust'
     'match': '\\b(ToOwned|ToString)\\b'
   }
   'Type': {
     'comment': 'A type'
     'name': 'entity.name.type.rust'
     'match': '\\b[A-Za-z]+\\b'
   }
```
And add after 161 line:
```cson
{ 'include': '#type' }
```